engine/internal/runtime/client.go

```diff
@@ -373,6 +373,25 @@ func (c *commonClient) deployRuntime(
 	if p := params.runtimePort; p != 0 {
 		cport = p
 	}
+
+	cmd := fmt.Sprintf(`
```

Comment:
Can't we do this only for vLLM? We can add a new parameter to `deployRuntimeParams` and set the field only in the `vLLMClient`.